src/win/pty.cc

```diff
@@ -126,6 +126,15 @@ static std::wstring get_shell_path(std::wstring filename)  {
   return shellpath;
 }
 
+void throw_winpty_error(const char *generalMsg, winpty_error_ptr_t error_ptr) {
+  std::stringstream why;
+  std::wstring msg(winpty_error_msg(error_ptr));
+  std::string msg_(msg.begin(), msg.end());
```

Comment:
I'm not 100% sure, but I think the initializer of `msg_` converts the wide string to a narrow string by truncating each wchar_t value to a char.  It will only be correct for ASCII characters.  I'm guessing node.js and/or v8 want UTF-8 strings?  Standard C++ has a way to do that conversion (as of C++11, I think -- I'm thinking of std::codecvt), and maybe node.js / v8 also does?  Windows has C-style APIs to do the conversion.

I think the pattern already existed in the code, so it's probably not necessary to fix it in this PR?  e.g. see `shellpath_`'s initializer.